webrtc/stun/stun_connection.nim

```diff
@@ -28,6 +28,7 @@ type
   StunUsernameProvider* = proc(): string {.raises: [], gcsafe.}
   StunUsernameChecker* = proc(username: seq[byte]): bool {.raises: [], gcsafe.}
   StunPasswordProvider* = proc(username: seq[byte]): seq[byte] {.raises: [], gcsafe.}
+  StunConnOnClose* = proc() {.raises: [], gcsafe.}
```

Comment:
It can be only `OnClose`, it is clear from the context it is related to a Stun conn.